### PR TITLE
Show platform, backend and capacity as badges in agent list

### PR DESCRIPTION
**Pull request by lafriks**
Screenshots:

attels

attels

### CODE REVIEW

File: web/src/components/atomic/Badge.vue [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<template>
```
Comment (anbraten) [inline web/src/components/atomic/Badge.vue:1]: @lafriks Isn't there already a similar component for the badges in the pipeline side panel for matrix pipelines? Would be nice to re-use them. I am also not so sure if blue is the best fitting color.

**Review by anbraten — COMMENTED** [review]
Nice idea.

File: web/src/components/atomic/Badge.vue (hunk 1 above)
Comment (lafriks) [reply web/src/components/atomic/Badge.vue:1]: @anbraten had not used matrix pipelines didn't know there was already badges. Updated badge style to match one in pipelines and made it to also use new component

Comment (anbraten) [reply web/src/components/atomic/Badge.vue:1]: thanks

**Review by anbraten — APPROVED** [review]

**Pull request merged**